packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -870,19 +870,36 @@ function findLocalizePositions(
   return positions;
 }
 
-async function loadLocaleData(path: string, optimize: boolean): Promise<string> {
+async function loadLocaleData(path: string, optimize: boolean, es5: boolean): Promise<string> {
   // The path is validated during option processing before the build starts
   const content = fs.readFileSync(path, 'utf8');
 
-  // NOTE: This can be removed once the locale data files are preprocessed in the framework
-  if (optimize) {
-    const result = await terserMangle(content, {
-      compress: true,
-      ecma: 5,
-    });
+  // Downlevel and optimize the data
+  const transformResult = await transformAsync(content, {
+    filename: path,
+    // The types do not include the false option even though it is valid
+    // tslint:disable-next-line: no-any
+    inputSourceMap: false as any,
+    babelrc: false,
+    configFile: false,
+    presets: [
+      [
+        require.resolve('@babel/preset-env'),
+        {
+          bugfixes: true,
+          // IE 9 is the oldest supported browser
+          targets: es5 ? { ie: '9' } : { esmodules: true },
```

Comment:
Shouldn't we use `targets` derived from browserslist, similar to the downlevelling process?


https://github.com/angular/angular-cli/blob/6c8b0f131595edcc278e14a4723ff01fd4419279/packages/angular_devkit/build_angular/src/utils/process-bundle.ts#L129-L164

Reply:
thoughts @clydin ?

Reply:
I actually copied the code from there and simplified it.  Went with this approach to reduce the complexity and make the output more deterministic.  When the `es5` parameter is true, the content returned is required to be es5 compatible.  Targeting IE9 guarantees that the output code will work for all supported browsers that require ES5.  And since the content is well-known and controlled and essentially an array of numbers and strings, there's really not a tangible benefit to targeting IE11 over IE9.
On the other side of the `es5` conditional, this would be possible but the targets would need to be scrubbed of any older browsers.  However, the combination of the `bugfixes` option and the `esmodules` option causes only a minimal set of changes to code (mainly to fix engine bugs) and allows most ES2016/ES2017 features to remain that would otherwise be removed.  This part can also be revisited in the future if even newer features are introduced into the locale data but right now it doesn't appear that the output would benefit.
This approach also eliminates any odd behavior due to a broken browserslist file or defects in the browserslist/caniuse packages.